float64 data_front
float64 data_left
float64 data_right
